skillful_network_client/src/app/registration-confirmation/registration-confirmation.component.html

```diff
@@ -0,0 +1,54 @@
+<div class="container">
+    <mat-card class="connectionForm">
+        <div class="logo">
+            <img fxLayoutAlign="center center" class="logoUniversite" src="assets/pictures/Logo_complet.png"
+                alt="logo université">
+        </div>
+        <mat-card-title fxLayoutAlign="center center">Finalisation inscription
+            <i class="material-icons">
+                perm_identity
+            </i>
+        </mat-card-title>
+        <mat-card-content>
+            <form [formGroup]="registrationForm" fxLayout="column" fxLayoutAlign="center center" novalidate
+                (submit)="registration()">
+                <mat-form-field>
+                    <mat-label>Vous êtes: </mat-label>
+                    <mat-select formControlName="selectRole" (selectionChange)="getSelected($event)" (selectionChange)="setValidator($event)" >
+                        <mat-option *ngFor="let role of roles" [value]="role">
+                            {{role.valueView}}
+                        </mat-option>
+                    </mat-select>
+                </mat-form-field>
+                <!--NAME-->
+                <mat-form-field>
+                    <mat-label>Nom</mat-label>
+                    <input formControlName="name" matInput placeholder="{{registrationForm.value.selectRole.valueView}}"
+                        class="form-control">
+                    <mat-icon *ngIf="registrationForm.value.selectRole.value != 'ROLE_USER'" matSuffix>business
+                    </mat-icon>
+                    <mat-icon *ngIf="registrationForm.value.selectRole.value == 'ROLE_USER'" matSuffix>face</mat-icon>
+                    <mat-error
+                        *ngIf="registrationForm.get('name').hasError && registrationForm.get('name').dirty || registrationForm.get('name').touched">
+                        <span *ngIf="registrationForm.get('name').hasError('required')">Votre nom est
+                            <strong>requis</strong></span>
+                    </mat-error>
+                </mat-form-field>
+                <!--FIRST NAME-->
+                <mat-form-field *ngIf="registrationForm.value.selectRole.value == 'ROLE_USER'">
+                    <mat-label>Prénom</mat-label>
+                    <input formControlName="firstName" matInput
+                        placeholder="{{registrationForm.value.selectRole.valueView}}" class="form-control">
+                    <mat-icon matSuffix>face</mat-icon>
```

Comment:
Use `<i class="material-icons md-24"> face</i>` instead of `mat-icon`. The goal is to not duplicate style code, and we will be able to adapt the style consistently.

Reply:
Do that for all the `mat-icon` maybe?

Reply:
Could you explain why it's more interesting to use <i> instead of  <mat-icon>?
We reuse angular material module (svg icon) and add CSS with :

mat-icon.size-16 {   
  width: 16px;
  height: 16px;
}
<mat-icon class="size-16">

It's not the same thing?
Thanks!